ceph-dev-new-trigger/config/definitions/ceph-dev-new-trigger.yml

```diff
@@ -28,20 +28,47 @@
           wipe-workspace: true
 
     builders:
-      - shell:
-          !include-raw:
-            - ../../../scripts/build_utils.sh
-            - ../../build/notify
-      - trigger-builds:
-        - project: 'ceph-dev-new'
-          predefined-parameters: |
-            BRANCH=${GIT_BRANCH}
-            FORCE=True
-        - project: 'ceph-dev-new'
-          predefined-parameters: |
-            BRANCH=${GIT_BRANCH}
-            FORCE=True
-            FLAVOR=notcmalloc
```

Comment:
I know this is already merged, but it seems that `FLAVOR=notcmalloc` was not kept with the addition. This means identical builds

Reply:
@alfredodeza fixed at #938.